content/en/docs/ops/best-practices/observability/index.md

```diff
@@ -0,0 +1,268 @@
+kind: ServiceMonitor
+metadata:
+  name: istio-federation
+  labels:
+    app.kubernetes.io/name: istio-prometheus
+spec:
+  namespaceSelector:
+    matchNames:
+    - istio-system
+  selector:
+    matchLabels:
+      app: prometheus
+  endpoints:
+  - interval: 30s
+    scrapeTimeout: 30s
+    params:
+      'match[]':
+      - '{__name__=~"workload:(.*)"}'
+      - '{__name__=~"pilot(.*)"}'
+    path: /federate
+    targetPort: 9090
+    honorLabels: true
+    metricRelabelings:
+    - sourceLabels: ["__name__"]
+      regex: 'workload:(.*)'
+      targetLabel: "__name__"
+      action: replace
+{{< /text >}}
+
+{{< tip >}}
+The key to the federation configuration is matching on the job in the Istio-deployed Prometheus that is collecting
+[Istio Standard Metrics](/docs/reference/config/telemetry/metrics/) and renaming any metrics collected by removing
+the prefix used in the workload-level recording rules (`workload:`). This will allow existing dashboards and
+queries to seamlessly continue working when pointed at the production Prometheus instance (and away from the Istio instance).
+
+Control plane metrics are also collected and federated up to the production Prometheus.
+{{< /tip >}}
+
+### Optimizing metrics collection with recording rules
+
+Beyond just using recording rules to [aggregate over pods and instances](#workload-level-aggregation-via-recording-rules), you may
+want to use recording rules to generate aggregated metrics tailored specifically to your existing dashboards and alerts. Optimizing
+your collection in this manner can result in large savings in resource consumption in your production instance of Prometheus, in
+addition to faster query performance.
+
+For example, imagine a custom monitoring dashboard that used the following Prometheus queries:
+
+  * Total rate of requests averaged over the past minute by destination service name and namespace
+
+    {{< text json >}}
+    sum(irate(istio_requests_total{reporter="destination"}[1m]))
+    by (
+        destination_canonical_service,
+        destination_workload_namespace
+    )
+    {{< /text >}}
+
+  * P95 client latency averaged over the past minute by source and destination service names and namespace
+
+    {{< text json >}}
+    histogram_quantile(0.95,
+      sum(irate(istio_request_duration_milliseconds_bucket{reporter="source"}[1m]))
+      by (
+        destination_canonical_service,
+        destination_workload_namespace,
+        source_canonical_service,
+        source_workload_namespace,
+        le
+      )
+    ) >= 0
+    {{< /text >}}
+
+The following set of recording rules could then be added to the Istio Prometheus configuration, using the `istio` prefix
+to make identifying these metrics for federation simple.
+
+{{< text yaml >}}
+groups:
+- name: "istio.recording-rules"
+  interval: 30s
+  rules:
+  - record: "istio:istio_requests:by_destination_service:rate1m"
+    expr: |
+      sum(irate(istio_requests_total{reporter="destination"}[1m]))
```

Comment:
You're using `destination` here and `source` on the example below without explaining why?  In the interests of simplicity couldn't you use `source` for both.

Reply:
Sounds good. moved to `source` for both.